.github/workflows/release.yaml

```diff
@@ -20,4 +20,9 @@ jobs:
     - name: "create the release in GitHub"
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      run: ./.ci/create-release-github.sh
+      run: |
+        OPERATOR_VERSION=$(git describe --tags)
+        gh release create \
+            -t "Release ${OPERATOR_VERSION}" \
+            "${OPERATOR_VERSION}" \
+            'dist/tempo-operator.yaml#Installation manifest for Kubernetes' \
```

Comment:
Which manifests get uploaded? Only the k8bernetes one and not the openshift?

I am fine to publish only the k8s one, on openshift let people use the OLM.